backendless_chef.yaml

```diff
@@ -305,6 +312,18 @@ Resources:
         IpProtocol: tcp
         SourceSecurityGroupId: !Ref SSHSecurityGroup
         ToPort: '22'
+      - FromPort: '10000'
```

Comment:
Why not combine these, for simplicity, to a single rule with `FromPort: '10000'` and `ToPort: '10003'` ?